packages/flutter_tools/lib/src/commands/test.dart

```diff
@@ -100,6 +100,13 @@ class TestCommand extends FastFlutterCommand {
         allowed: const <String>['tester', 'chrome'],
         defaultsTo: 'tester',
         help: 'The platform to run the unit tests on. Defaults to "tester".',
+      )
+      ..addOption('test-randomize-ordering-seed',
+        defaultsTo: '0',
+        help: 'If positive, use this as a seed to randomize the execution of '
+              'test cases (must be a 32bit unsigned integer).\n'
```

Comment:
That seems oddly specific, I assume this is for JavaScript compatibility?

Reply:
Probably?  It's the same copy as from package:test

Reply:
ACK